boards/Kconfig

```
@@ -2708,6 +2717,7 @@ config ARCH_BOARD
 	default "esp32c3-devkit-rust-1"    if ARCH_BOARD_ESP32C3_DEVKIT_RUST1
 	default "esp32s2-kaluga-1"         if ARCH_BOARD_ESP32S2_KALUGA_1
 	default "esp32s2-saola-1"          if ARCH_BOARD_ESP32S2_SAOLA_1
+	default "esp32s2-franzininho"      if ARCH_BOARD_ESP32S2_FRANZININHO
```

```suggestion
	default "franzininho-wifi"      if ARCH_BOARD_ESP32S2_FRANZININHO
```
Rename board name for consistency.